config/maintainers.json

```diff
@@ -20,6 +20,16 @@
       "link_url": null,
       "avatar_url": null,
       "bio": "Front-end web developer with years of experience using JavaScript, but learning a functional approach to web development using PureScript and Haskell"
+    },
+    {
+      "github_username": "ferhatelmas",
+      "alumnus": false,
+      "show_on_website": true,
```

Comment:
```suggestion
      "show_on_website": false,
```

Reply:
I copied my config on go, should I update there too?

Reply:
No, you've been around Go long enough to be considered a "Senior Maintainer" there :)

The aim with this is that people contribute to a track a bit **before** they get the same credit on the website that other maintainers on that track get.

Reply:
Sounds good, thanks!